src/main/java/com/nextm3project/dtos/TruckDto.java

```diff
@@ -0,0 +1,50 @@
+package com.nextm3project.dtos;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.Size;
+
+//Criado para receber e fazer as validações dos dados recebidos,
+//pois se tiver dados errados, devem ser descartados e enviar uma bad request para o cliente.
+
+public class TruckDto {
+
+		
+	@NotBlank							//Verifica se o campo não está nulo, se tem string vazia e etc.
+	@Size(max = 7)						//Verifica se tem o máximo de 7 caracteres
+	private String licensePlateTruck;
+	
+	@NotBlank
+	@Size(max = 5)
+	private String status;
```

Comment:
Geralmente Status a gente lida com tipo boolean, ou int enum.

Dêem uma olhada aqui: https://www.baeldung.com/a-guide-to-java-enums
Aqui tem um guia de como criar uma classe enum e utilizá-la.